aiida/orm/utils/loaders.py

```diff
@@ -121,6 +122,39 @@ def load_code(identifier=None, pk=None, uuid=None, label=None, sub_classes=None,
     )
 
 
+def load_container_code(
```

Comment:
Don't think we need this indeed. For most use-cases `load_code` will work (provided that the entry point becomes `core.code.something`.

That being said, it will share the same uniqueness restrictions as all other codes. What I mean is that since there is no uniqueness criterion on the code label, it is possible to create two codes with the same label. This is currently a problem in AiiDA, and one we decided not to fix (`verdi code setup` will not allow it, but the ORM will allow it). This means that `load_code` on that label will fail. But this will also fail for a `Code` and `ContainerizedCode` with the same label, and maybe that is more unexpected for users. I don't think it is a problem as this already exists, but I just thought I'd mention it here.